app/javascript/components/common/svg-graph-util.ts

```diff
@@ -7,119 +7,221 @@ export type Line = {
   length: number
   angle: number
 }
-
-const SMOOTHING_RATIO = 0.2
+const SIMILAR_THRESHOLD_DELTA = 0.01
+const SMOOTHING_THRESHOLD_DELTA = 0.2
```

Comment:
If points are more than 20% different from eachother (with respect to the total height of the graph) they will be considered extreme enough to be smoothed